src/components/actions/Button/Button.module.css

```
@@ -0,0 +1,165 @@
+  border-color: var(--zp-color-danger-hover);
+}
+.variant-danger:active:not(:disabled):not([aria-disabled='true']) {
+  background-color: var(--zp-color-danger-active);
+  border-color: var(--zp-color-danger-active);
+}
+
+/* Loading spinner */
+.spinner {
+  width: 1em;
+  height: 1em;
+  border: 2px solid currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)